### PR TITLE
Analytics: Restore property in export json event

### PR DESCRIPTION
**Pull request by evictorero**
**What is this feature?**

Fixing missing properties after [this refactor](https://github.com/grafana/grafana/pull/79078)

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/features/dashboard/utils/getPanelChromeProps.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ export function getPanelChromeProps(props: CommonProps) {
 
   const onCancelQuery = () => {
     props.panel.getQueryRunner().cancelQuery();
-    DashboardInteractions.panelCancelQueryClicked();
```
Comment (juanicabanas) [inline public/app/features/dashboard/utils/getPanelChromeProps.tsx:82]: why do we need this?

Comment (evictorero) [reply public/app/features/dashboard/utils/getPanelChromeProps.tsx:82]: I am assuming we want it, but we lost it in the refactor like the others, but tbh, I don't know.

Comment (juanicabanas) [reply public/app/features/dashboard/utils/getPanelChromeProps.tsx:82]: I would just keep it as it is right now. We're not sure

Comment (evictorero) [reply public/app/features/dashboard/utils/getPanelChromeProps.tsx:82]: Ok, maybe @dprokop can help us here. 
Was [this change](https://github.com/grafana/grafana/pull/79078/files#diff-2e4d7a90c2dcc990dd68e1cc1e882ff084d676c94bc5f70f4a78424a2bd25bb9L81) intended or is a regression?
I found that now we are calling this event from scenes [here](https://github.com/grafana/grafana/blob/main/public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts#L511) so we should update both if we decide to restore it.

Comment (dprokop) [reply public/app/features/dashboard/utils/getPanelChromeProps.tsx:82]: @evictorero - it's a regression, sorry!

**Review by juanicabanas — APPROVED** [review]

File: public/app/features/dashboard/utils/getPanelChromeProps.tsx (hunk 1 above)
Comment (juanicabanas) [reply public/app/features/dashboard/utils/getPanelChromeProps.tsx:82]: This is ok to be modified then. @evictorero Just approved

**Pull request merged**